### ISSUE TITLE
Profile page: Ensure aria labels are unique in social nav

### ISSUE THREAD

**Issue by mejiaj** [Bixal/bixal-site-drupal#98]
Ensure there aren't duplicate aria labels with multiple social navs.

![image](https://github.com/Bixal/bixal-site-drupal/assets/3385219/56e18915-06da-46ca-b7a7-976e0803f357)

```
Landmarks should have a unique role or role/label/title (i.e. accessible name) combination

To solve this problem, you need to fix the following:
The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

Related Node
<nav class="bix-social-nav " aria-label="Social links">
```
